runtime/src/main/java/org/corfudb/util/NetworkUtils.java

```diff
@@ -2,9 +2,18 @@
 
 import java.net.Inet4Address;
 import java.net.InetAddress;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.net.InetAddress'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)